domains/cvargasgt.is-a.dev.json

```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "Carlosvva",
+        "email": "cv.1411.vargas@gmail.com",
+        "discord": "cvargasgt"
+    },
+    "record": {
+        "TXT": "vc-domain-verify=cvargasgt.is-a.dev,5500e44ef7fc1178b301"
```

Comment:
Invalid for vercel. See vercel docs for more information: https://docs.is-a.dev/guides/vercel